tests/test_experiment.py

```diff
@@ -17,6 +12,44 @@
 from mozanalysis.segments import Segment, SegmentDataSource
 
 
+@pytest.fixture()
+def desktop_metrics():
```

Comment:
Seems like this functionality is defined in a bunch of test files. How about placing it in a single file and then all the test files can import?

Reply:
Added in `helpers/config_loader_lists.py`